test/common/src/main/kotlin/org/codice/compliance/verification/ResponseVerifier.kt

```diff
@@ -30,6 +28,9 @@ import org.codice.security.sign.Decoder.InflationException.InflErrorCode
 import org.w3c.dom.Node
 import java.io.IOException
 
+/* ktlint-disable no-wildcard-imports */
+import org.codice.compliance.SAMLSpecRefMessage.*
+
```

Comment:
❓ Are we not able to import what we need without using wildcard for the `SAMLSpecRefMessage` stuff?

Reply:
Wildcard imports are not allowed under ktlint so we need a suppressor

Reply:
Wildcards are disallowed in our Java standard; let's be consistent.

https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

Reply:
Yeah, that was a question more for @brjeter on whether or not we can have individual import instead of suppressing it. If doing the individual is too much, I'm okay with the suppression.

Reply:
If it's too many one thing you can do is import `org.codice.compliance.SAMLSpecRefMessage` instead and of using `SAMLCore_3_2_2_a` in the code do `SAMLSpecRefMessage.SAMLCore_3_2_2_a`

Reply:
I'd rather not do that ☝️ I'm fine with the suppression or, to Richard's point, just importing each individual message. Developers rarely look at the import blocks.